docs/Measuring_Resonances.md

```diff
@@ -42,14 +80,18 @@ Recommended connection scheme for I2C:
 | SDA | 03 | GPIO02 (SDA1) |
 | SCL | 05 | GPIO03 (SCL1) |
 
+![MPU-9250 connected to RPI](img/mpu9250-PI-fritzing.png)
 
-Fritzing wiring diagrams for some of the ADXL345 boards:
-
-![ADXL345-Rpi](img/adxl345-fritzing.png)
+Recommended connection scheme for I2C(i2c0a) on the RP2040:
 
+| MPU-9250 pin | RP2040 pin | RPi pin name |
+|:--:|:--:|:--:|
+| 3V3 (or VCC) | 39 | VSYS |
```

Comment:
I don't think most MPU boards come with a 3.3v pin. They all have a little LDO that can take 5v and the input is marked as VCC. I think some errors happened when I was making the chart. Is there any particular reason to use 3.3v out v.s. the 5v?

Reply:
If the MPU board has level shifters and you connect it to 5V you have a good chance to kill the host since most host boards are not 5v tolerant. In addition you cannot rely on the fact that there is always a LDO

Board without LDO:
https://www.sparkfun.com/products/11028

There are ADXLs with Level Shifters, don't know if there are MPU boards with them as well.

Reply:
Yeah, as Sineos said, the main worry was about frying either an accelerometer or a Pi in case of voltages mismatches. If an MPU board has a logic level shifter, RPi may be damaged, if an MPU doesn't have an LDO, the accelerometer may get damaged. An example of a board with the logic level shifters (on some pins):
https://learn.adafruit.com/mpu6050-6-dof-accelerometer-and-gyro/downloads
So, thanks for updating, now it looks good.